### PR TITLE
python/python3-tekore

### PR DESCRIPTION
**Pull request by RSKYS**

### CODE REVIEW

**Comment by RSKYS** [comment]
@sbo-bot: build python/python3-tekore

**Comment by RSKYS** [comment]
@sbo-bot: build python/python3-tekore

File: python/python3-tekore/python3-tekore.info [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-VERSION="4.3.0"
+VERSION="5.4.0"
 HOMEPAGE="https://pypi.org/project/tekore/"
-DOWNLOAD="https://files.pythonhosted.org/packages/60/35/faea7eb3e936eeb30edb196245ce280512139c084afd5245a6fe3cbaa477/tekore-4.3.0.tar.gz"
-MD5SUM="d58ba00cd9271d97c926b52b257a158e"
+DOWNLOAD="https://files.pythonhosted.org/packages/18/f4/c4e4bcd154c122a5cabb6f64490cfa036685c8c76495ff465a876bf3f04e/tekore-5.4.0.tar.gz"
+MD5SUM="93287215d4af5d9b40cd29db0b345be1"
 DOWNLOAD_x86_64=""
 MD5SUM_x86_64=""
 REQUIRES=""
```
Comment (willysr) [inline python/python3-tekore/python3-tekore.info:8]: You need to adjust the deps to use new method to build with wheel

**Review by willysr — CHANGES_REQUESTED** [review]
please update the dep

**Comment by RSKYS** [comment]
I see.
We need python3-build and python3-wheel in this right?

**Comment by willysr** [comment]
yes

**Comment by willysr** [comment]
@sbo-bot: build python/python3-tekore

**Comment by willysr** [comment]
it seems you tested this under current?

**Comment by RSKYS** [comment]
> it seems you tested this under current?

I mean, unfortunately it doesn't give me much of options, because new sources haven't got setup.py either way

**Comment by RSKYS** [comment]
Is there anyway I can use sbo-bot whether on this repo or my own, or it has to be self-hosted?

**Comment by RSKYS** [comment]
@sbo-bot: build python/python3-tekore

**Comment by willysr** [comment]
@sbo-bot: build python/python3-tekore

**Comment by willysr** [comment]
you need to use python3-setuptools-opt instead

**Comment by RSKYS** [comment]
@sbo-bot: build python/python3-tekore

File: python/python3-tekore/python3-tekore.info [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+VERSION="5.1.1"
 HOMEPAGE="https://pypi.org/project/tekore/"
-DOWNLOAD="https://files.pythonhosted.org/packages/60/35/faea7eb3e936eeb30edb196245ce280512139c084afd5245a6fe3cbaa477/tekore-4.3.0.tar.gz"
-MD5SUM="d58ba00cd9271d97c926b52b257a158e"
+DOWNLOAD="https://github.com/felix-hilden/tekore/archive/v5.1.1/tekore-5.1.1.tar.gz"
+MD5SUM="12250450c0584df39513439d5e8e52f5"
 DOWNLOAD_x86_64=""
 MD5SUM_x86_64=""
-REQUIRES=""
+REQUIRES="python3-setuptools-opt"
```
Comment (aclemons) [inline python/python3-tekore/python3-tekore.info:8]: You need to actually change your slackbuild to use the setuptools installed in /opt.

For example:

https://github.com/SlackBuildsOrg/slackbuilds/blob/ee8422d3d2d914051f468983ccf81c2dd474791d/python/python3-tempora/python3-tempora.SlackBuild#L73-L74

Comment (RSKYS) [reply python/python3-tekore/python3-tekore.info:8]: Hello!
Let me try that. thanks.

**Comment by aclemons** [comment]
@sbo-bot: build i586 python/python3-tekore

**Comment by aclemons** [comment]
@sbo-bot: build x86_64 python/python3-tekore

**Comment by RSKYS** [comment]
Check this one using sbo-bot too, if it was okay, merge both
If not, only merge first one
Thanks

**Comment by willysr** [comment]
@sbo-bot: build x86_64 python/python3-tekore

**Comment by willysr** [comment]
merged

**Pull request closed without merging**